### DISCUSSION TITLE
Good First Issues?

### DISCUSSION THREAD

**Discussion by ndungudedan** [ACINQ/lightning-kmp#614] [Developers]
Hello!

I am new to the Eclair project and looking for good first issues to tackle. Please feel free to suggest such issues mostly on lightning-kmp, bitcoin-kmp, or the Phoenix app.

**Comment by t-bast** [comment]
Welcome! To be honest I don't know if we have good issues or small features to work on at the moment...we're rather working on big features (splicing, liquidity ads, taproot, bolt 12) which aren't well suited for new contributors. But don't hesitate to poke around to see if you find things you'd like to work on!

**Reply by ndungudedan** [reply]
I did find this: [TODO: If some feerates are null, we may implement a retry](https://github.com/ACINQ/lightning-kmp/blob/e5da4489c3bdc85bdce74634032c7df4a399639d/src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt#L266). I'm thinking doing a time based delay (maybe 10 seconds) and then retrying the fee rates estimation will work. Please feel free to let me know your idea, as I tackle this.
@t-bast
